src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql

```diff
@@ -1,4 +1,15 @@
 with
+    schools as (
+        select
+            _dbt_source_relation,
```

Comment:
remove: it only appears on kipptaf tables

Reply:
done!